rockcraft.yaml

```diff
@@ -0,0 +1,52 @@
+name: ratings
+version: 0.0.4
+license: GPL-3.0
+
+base: bare
+build_base: ubuntu@24.04
+platforms:
+  amd64:
+
+summary: Ubuntu App Ratings Service
+description: |
+  Backend service to support application ratings in the new Ubuntu Software Centre.
+run_user: _daemon_
+
+environment:
+  APP_ENV: dev
+  APP_HOST: 0.0.0.0
+  APP_LOG_LEVEL: info
+  APP_NAME: ratings
+  APP_PORT: 18080
+  APP_JWT_SECRET: "foo"
```

Comment:
I'm assuming this is a placeholder, but can we have a comment explaining how this is used / overwritten when deployed please?

Reply:
I can't actually remember why I left in placeholder originally. I inject a number of ENV vars in the Charm (including most of these), and will know before I run the container all of these variables I need. Let me remove them for now, move this to the Charm and leave out placeholders for readability.